src/torrent_info.cpp

```diff
@@ -539,16 +547,21 @@ namespace {
 	}
 
 	int load_file(std::string const& filename, std::vector<char>& v
-		, error_code& ec)
+		, error_code& ec, int const max_buffer_size = 80000000)
```

Comment:
If `max_buffer_size` is forced to this value here, what is the point of setting the default to `6000000` in `torrent_info.hpp:119`? Also, is this supposed to be the file size limit in bytes? It should be at least configurable, like the piece limit and such.

Reply:
it's not forced, it's just the default, for all the calls to this function that doesn't specify it. It is configurable via that `load_torrent_limits::max_buffer_size`

Reply:
Ok, understood.